src/clusterfuzz/_internal/bot/fuzzers/syzkaller/runner.py

```diff
@@ -70,7 +81,32 @@ def get_config():
 
 def get_cover_file_path():
   """Return location of coverage file for Syzkaller."""
-  return os.path.join(get_work_dir(), 'coverfile')
+  file_path = os.path.join(get_work_dir(), 'coverfile')
+
+  logs.log(f'Rawcover file will be stored in {file_path}')
```

Comment:
nit: a better place for this log call would be the caller (i.e. line 210).  Otherwise if in the future this gets called multiple times we'll get redundant logs.